Code/Framework/AzCore/AzCore/Serialization/EditContextConstants.inl

```diff
@@ -194,6 +194,9 @@ namespace AZ
             //! For use with slice creation tools. See SliceCreationFlags below for details.
             const static AZ::Crc32 SliceFlags = AZ_CRC("SliceFlags", 0xa447e1fb);
 
+            //! To allow the transform components lineedit to be modified dependant on position in the heirarchy.
+            const static AZ::Crc32 TransformParentClearButton = AZ_CRC_CE("TransformParentClearButton");
```

Comment:
Even with this description, I'm not sure what this attribute is doing honestly, can you explain?

Also, nit: heirarchy -> hierarchy